apps/api/src/app/execution-details/execution-details.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { USE_CASES } from './usecases';
+import { SharedModule } from '../shared/shared.module';
+import { AuthModule } from '../auth/auth.module';
+
+@Module({
+  imports: [SharedModule, AuthModule],
+  providers: [...USE_CASES],
+  controllers: [],
+})
+export class ExecutionDetailsModule {}
```

Comment:
@p-fernandez @davidsoderberg do you think we need a module for this one, or we can also have it living in the existing activity-feed module? Since on it's own the execution details doesn't feel to me like a domain of it's own. Wdyt?

Reply:
I think that events are also a good place for it. The way I viewed it is that on its own, without the other "Notification" entity, we don't have the full picture for this particular domain. So co-locating them makes sense to me in that scenario. I would have it in the events module rather than in a standalone one. But this is something we could quite easily transfer later if we will see the need to. So feel free to make the call here.

Reply:
@scopsy I think it is nice to have it in its own module to be able to inject the module in different modules and keep it as small as possible 😄

Reply:
@scopsy, @davidsoderberg and me agreed to have its own separated module now for the reasons he mentioned and for simplicity. In the near future seeing how the domain evolves we can have a better idea where to let it live and move it if needed.